src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -137,20 +142,20 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
 
   onMutationError(error, title?, message?) {
     logger.error(error)
-    this.props.dialogs.showErrorDialog({ title, message })
+    showErrorDialog(this.props.dialog, { title, message })
     this.setState({ isCommittingMutation: false })
   }
 
   confirmOfferTooLow() {
-    return this.props.dialogs.showAcceptDialog({
+    return showAcceptDialog(this.props.dialog, {
       title: "Offer may be too low",
       message:
         "Offers within 20% of the list price are most likely to receive a response.",
     })
   }
 
   confirmOfferTooHigh() {
-    return this.props.dialogs.showAcceptDialog({
+    return showAcceptDialog(this.props.dialog, {
```

Comment:
@ds300 - Following convention it seems like `injectDialog` would make these functions accessible via props (as they were before). I can see a dev in the future using `injectDialog` but then being confused about how to trigger behavior.

Reply:
Also, it seems like a bit of indirection to pass an injected `dialog` prop into an imported function to display. Since we're using `injectDialog` it would be best to keep everything things constrained around a single point: 
```js
this.props.showAcceptDialog({
  title: 'Offer higher'
  ...
})
```
Otherwise there's a bit too much juggling.